test/parallel/test-http-set-timeout-server.js

```diff
@@ -117,20 +117,25 @@ test(function serverRequestNotTimeoutAfterEnd(cb) {
 
 test(function serverResponseTimeoutWithPipeline(cb) {
   let caughtTimeout = '';
+  let secReceived = false;
   process.on('exit', function() {
     assert.strictEqual(caughtTimeout, '/2');
   });
   const server = http.createServer(function(req, res) {
```

Comment:
may be worthwhile adding in `common.mustCall()` for the various callbacks in this test.